src/card.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default function Card(props) {
```

Comment:
The name `Card` is misleading. It suggests that this component can only render one "card".